src/servers/event-handlers/bria.ts

```diff
@@ -160,6 +132,24 @@ export const payoutSettledEventHandler = async ({
   event: PayoutSettled
 }): Promise<true | ApplicationError> => {
   const res = await Wallets.settlePayout(event.id)
+  return handlePayoutHandlerResult(res)
+}
+
+const handleIncomingUtxoHandlerResult = (res: true | ApplicationError) => {
```

Comment:
I think we discussed one time this distinction actually belongs in the use case - not in the top level handler.

Reply:
> I think we discussed one time this distinction actually belongs in the use case - not in the top level handler.

Yea true. I think I'll close this one for now then and tackle that as a separate thing the next time it comes up. Don't want to further normalize this sort of handling here.